04/style.css

```diff
@@ -0,0 +1,91 @@
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+body{
+    background-color: rgb(105, 105, 105);
+}
+a{
+    text-decoration: none;
+    color: #000;
+}
+.navigation{
+    width: 100%;
+    background-color: #fff;
+}
+.navigation li{
+    padding: 10px;
+    text-transform: uppercase;
+}
+.navigation ul{
+    list-style-type: none;
+}
+.hamburger{
+    display: none;
+}
+
+@media(max-width: 600px){
+    .navigation{
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+    .navigation .hamburger{
+        align-self: flex-end;
+    }
+    .hamburger{
+    display: block;
+    width: 40px;
+    height: 35px;
+    border: 0;
+    border-top: 5px solid #000;
+    background: transparent;
+    position: relative;
+    transition:  0.3s transform linear;
+}
+.hamburger::before, .hamburger::after{
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    border-top: 5px solid #000;
+    transform: translateY(10px);
+}
+.hamburger::after{
+    transform: translateY(25px);
+     transition:  0.3s transform linear;
+}
+.hamburger:focus{
+    transform: rotate(45deg) translateY(5px);
+    border: none;
+}
+.hamburger:focus::after{
+    transform: rotate(-90deg)translateX(-10px);
+}
+.main__menu{
+    display: none;
+}
+.hamburger:focus + .main__menu{
+    display: block;
+}
+}
+@media(min-width:601px){
+    .navigation .main__menu{
+        display: flex;
+    }
+    .second__list{
+        position: relative;
+        display: none;
+        visibility: hidden;
+    }
+    #second__listKey:focus + .second__list{
+        position: absolute;
+        display: block;
+        visibility: visible;
+        top: 35px;
```

Comment:
Wartość dla `top` jest taka sama jak w `height` - warto to wrzucić do zmiennej bo te wielkości zależą od siebie.
Jak zmienisz w jednym miejscu tą wartość to dobrze, aby w drugim też się zmieniła.